tests/integration/test-k8s.py

```diff
@@ -0,0 +1,28 @@
+"""Copyright 2024 JasmineGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from time import sleep
+import test
+from kubernetes import client, config
+
+if __name__ == '__main__':
+    config.load_kube_config()
+
+    v1 = client.CoreV1Api()
+    while True:
+        try:
+            host = v1.read_namespaced_service(name='jasminegraph-master-service',
+                                              namespace='default').spec.cluster_ip
+            break
+        except Exception:
+            sleep(0.2)
```

Comment:
Can this sleep introduce unexpected behaviors?

Reply:
This is introduced to check whether the jasminegraph master service is started.
I think we can remove this since we have enabled the same check in `test-k8s.sh` file as well.
@thevindu-w WDYT?

Reply:
Removed sleep as it is expected to have the jasminegraph master service running when the script is executed

Reply:
Ok